js/asyon.js

```diff
@@ -1,35 +1,54 @@
+                 'ajit':         ['körükleme','çırpıntı'],
+                 'anim':         ['canlandırma'],
+                 'aplik':        ['uygulama', 'yazılım'],
+                 'dejener':      ['yozlaşma', 'soysuzlaşma', 'bozunum'],
+                 'destin':       ['hedef', 'varış noktası', 'gidilecek yer'],
+                 'dokumant':     ['belgelendirme', 'yazıya dökme'],
+                 'dramatiz':     ['canlandırma'],
+                 'enform':       ['bilgi', 'malûmat'],
+                 'entegr':       ['bütünleşme', 'uyum'],
+                 'form':         ['biçimlenme', 'yetişim'],
+                 'halusin':      ['sanrı'],
+                 'idealiz':      ['ülküleştirme'],
+                 'implement':    ['uyarlama', 'gerçekleştirme'],
+                 'izol':         ['yalıtım'],
+                 'jener':        ['nesil'],
+                 'kalifik':      ['beceri', 'yetenek', 'tecrübe', 'niteliklilik'],
+                 'komunik':      ['iletişim'],
+                 'konfigur':     ['yapılandırma', 'ayarlama'],
+                 'konfirm':      ['doğrulama', 'onaylama', 'geçerleme'],
+                 'konsantr':     ['derişim', 'dikkat toplaşımı', 'yoğunlaşma'],
+                 'koordin':      ['eş güdüm'],
+                 'lok':          ['konum', 'mevki', 'yer', 'mahal'],
+                 'lokaliz':      ['yerelleştirme', 'kuruluş yeri seçimi'],
+                 'migr':         ['göç'],
+                 'oblig':        ['yükümlülük', 'zorunluluk', 'şart', 'borç'],
+                 'oksid':        ['yükseltgenme'],
+                 'oper':         ['ameliyat', 'müdahale', 'harekât'],
+                 'oryant':       ['yönlendirme', 'uyumlanma', 'eğitme'],
+                 'otantik':      ['doğrulama', 'kimlik denetimi', 'belgeleme'],
+                 'polariz':      ['kutuplanma', 'polarma', 'ucaylanma'],
+                 'popül':        ['nüfus', 'ana kitle', 'varlık'],
+                 'prezant':      ['sunum', 'tanıtım'],
+                 'provok':       ['kışkırtma'],
+                 'rasyonaliz':   ['ussallaştırma', 'akla dayattırma', 'gerçeklendirme'],
+                 'rejener':      ['yenilenme'],
+                 'rekonfirm':    ['tekrar doğrulama', 'tekrar onaylama'],
+                 'replik':       ['ikileme', 'çoğaltma', 'kopyalama'],
+                 'restor':       ['yenileme', 'onarım'],
+                 'rot':          ['yer değiştirme', 'dönüştürme', 'çevirme'],
+                 'sans':         ['dalgalanma'],
+                 'senkroniz':    ['eşzamanlama', 'eşleme'],
+                 'spekul':       ['vurgun', 'saptırma'],
+                 'telekomünik':  ['uziletişim', 'uzakiletişim'],
+                 'transform':    ['dönüşüm'],
+                 'transport':    ['ulaşım', 'taşıma'],
+                 'valid':        ['onaylama', 'doğrulama', 'geçerleme'],
+                 'valü':         ['değerleme'],
```

Comment:
they shouldn't include unicode chars since I'm using them for subdomains as well, this is valid for `telekomünik` at line 45. they should be `valu` and `telekomunik`